src/pages/events/configure-commerce.md

```diff
@@ -30,20 +30,30 @@ You must configure Commerce to communicate with your project. Configuration incl
 
 1. Copy the contents of the `<workspace-name>.json` file into the **Adobe I/O Workspace Configuration** field.
```

Comment:
@jhadobe can we change this to something like "entire contents". Let's make it easier to understand that this means the entire file, as it can be a large file with several fields. This step is a bit confusing bc all the other copy/paste config steps focus on 1 particular field/value.

Reply:
@naomirobichaud in my opinion "entire contents" would be redundant, but i do not think it harms anything to modify the sentence.

Reply:
Sounds good - IMO it's worth specifying